lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_auth_client.rb

```diff
@@ -0,0 +1,62 @@
+require 'fastlane_core/ui/ui'
+module Fastlane
+  UI = FastlaneCore::UI unless Fastlane.const_defined?("UI")
+  module Auth
+    module FirebaseAppDistributionAuthClient
+      TOKEN_CREDENTIAL_URI = "https://oauth2.googleapis.com/token"
```

Comment:
We left the SCOPE, CLIENT_ID, etc in the login action since it uses all of those constants. The token was an error and has been changed so it should only be in one place now.

Reply:
It would seem more appropriate to me to have them defined here, and I'd argue we could move the login functionality into this module as well to have all the auth stuff in one place. That could be a followup, though.

Reply:
That makes sense this is a good conversation to have. I can see that everything in the auth client can fit inside the login action. We were originally thinking that action would be a way for the user to authenticate if they did not have another option. The two can be combined so thats a good point. I would recommend doing that in another PR to refactor some stuff.

Reply:
> everything in the auth client can fit inside the login action

I don't think anything in this module belongs in the Login action though. Possibly some auth-specific code in the action could be moved in here.

But yeah, another PR would fine but not necessary.